stories/Components/Molecules/FMViewPicture/FMViewPicture.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+import { FMViewPicture } from "@Molecules";
+
+import { DUMMY_COMPANY_IMAGE_URL } from "@Definitions/constants";
+
+const story = {
+  title: "Molecules/View Picture",
+  component: FMViewPicture,
+  argTypes: {
+    children: { control: "text" },
+  },
+};
+
+export default story;
```

Comment:
We are now use this convention. kindly update. Thanks
```suggestion
const meta: Meta = {
  title: "Molecules/View Picture",
  component: FMViewPicture,
  argTypes: {
    children: { control: "text" },
  },
};

export default meta;
```